src/app/components/LiveLabel/index.stories.tsx

```diff
@@ -22,25 +20,26 @@ const Component = ({
   variant,
   offScreenText,
   children,
-  ariaHidden = false,
 }: PropsWithChildren<Props>) => {
   return (
     <ThemeProvider service={service} variant={variant}>
       <ServiceContextProvider service={service} variant={variant}>
-        <LiveLabel
-          ariaHidden={ariaHidden}
-          liveText={services[service][variant].translations.media.liveLabel}
-          offScreenText={offScreenText}
-        >
-          {children}
-        </LiveLabel>
+        <LiveLabel offScreenText={offScreenText}>{children}</LiveLabel>
       </ServiceContextProvider>
     </ThemeProvider>
   );
 };
 
 const Wrapper = styled.div`
   position: relative;
+  a {
+    text-decoration: none;
+    border-bottom: none;
```

Comment:
Just checking if this is valid CSS?  I see 'medium' output in Chrome rather than 'none'?

<img width="210" alt="Screenshot 2023-10-18 at 15 35 41" src="https://github.com/bbc/simorgh/assets/1061401/3c84db2f-811a-447e-9467-7772368f092d">

Reply:
This is for storybook only and shouldn't be representative of the real component. We are currently discussing with the team whether to apply CSS style with the old method with styled components or change it to using the themes with emotion.

Storybook is also being odd with styles for me in Chrome and Firefix, fine in Safari, and doesn't look like how the real component looks on the page. Storybook is naughty.

Reply:
We have changed this to a valid border-bottom value. We haven't spent time on getting the css perfect as this is storybook only (and Chrome is being troublesome... so hard to work on it when I can't see what I am adding). This is fine for now as it won't affect the real component, and it is good enough for testing in storybook.

Reply:
The valid border-bottom value does now show up in the css inspector (it doesn't change to medium)